crates/containerd-shim-wasm/src/lib.rs

```diff
@@ -13,4 +13,7 @@ pub(crate) mod sys;
 #[cfg(any(test, feature = "testing"))]
 pub mod testing;
 
+#[cfg(test)]
+mod test;
+
```

Comment:
Can you add docs to differentiate "testing" and "test" mods? I think many people would find this confusing.